sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/interface.scala

```diff
@@ -188,10 +187,12 @@ case class CatalogTablePartition(
       map += ("Partition Parameters" -> paramsJson)
     }
 
-    map += ("Created Time" -> JString(new Date(createTime).toString))
+    map += ("Created Time" -> JString(
+      DateTimeUtils.microsToInstant(DateTimeUtils.millisToMicros(createTime)).toString))
```

Comment:
do we need to pass the session timezone?

Reply:
We have `Iso8601TimestampFormatter` (and `Iso8601DateFormatter`), can't you use it for formatting?